tests/sla_tests.js

```diff
@@ -282,8 +374,8 @@ describe('SLA Bridge and Channels Tester', function() {
     function validChecking() {
       setTimeout(function() {
         if (dialed[0] && dialed[0].wasAnswered && bridgeChannels.length !== 0 &&
-            isMixing && channel.wasAnswered) {
-              done();
+          isMixing && channel.wasAnswered) {
```

Comment:
I am not sure I am a fan of the 'was' verb in this variable. I think channel.answered would be sufficient.

Also, I keep seeing loads of state variables. I think that you should spend some time trying to consolidate these (if possible). 

The way to write beautiful code is to in fact, write less code. 

The less code you have, the fewer data paths you have to unwind in the event of fixing bugs or extending functionality. It is also much easier for a human to easily glean intent without having to execute the code through a debugger.